src/proof.ml

```diff
@@ -0,0 +1,44 @@
+open Core
+open! Lplib
+open Term
+open Timed
+open ConfigBuiltin
+
+(** [is_axim s excluded] return true if [s] ends with string `_ax` and [s] is 
+    different from [excluded]. *)
+let is_axiom : string -> string -> bool = fun s excluded ->
+    if String.length s > 3 && s <> excluded then
+        String.sub s (String.length s - 3) 3 = "_ax"
+    else
+        false
+        
+(** [output file_name sign proof signature package] create a file that contains
+    the deskolemized proof [proof] with the signature [signature]. *)
+let output : string -> Sign.t -> term -> string -> string -> unit = 
+    fun file_name sign proof signature package ->
+    (* get the config from the signature. *)
+    let cfg = get_config sign in

```

Comment:
Why not passing cfg as argument instead of sign so that get_config is called only once?